react-interactive/src/FilterableTable/DateTimeFilters.tsx

```diff
@@ -250,4 +254,122 @@ export function TimeFilter<T>(props: IProps<T>) {
             </tr>
         </> : null}
     </>
+}
+
+export function DateTimeFilter<T>(props: IProps<T>) {
+    const [dateTime, setDateTime] = React.useState<string>('');
+    const [secondDateTime, setSecondDateTime] = React.useState<string>('')
+
+    const val: IValue = React.useMemo(() => ({Value: dateTime}), [dateTime])
+    const val2: IValue = React.useMemo(() => ({Value: secondDateTime}), [secondDateTime])
+
+    const [operator, setOperator] = React.useState<FilterTypes>('after');
+
+    React.useEffect(() => {
+        if (props.Filter.length === 0) {
+            setDateTime('');
+            setSecondDateTime('');
+        }
+        if (props.Filter.length > 1) {
+            const f1 = props.Filter.find(f => f.Operator === '>' || f.Operator === '>=')
+            if (f1 == null)
+                setDateTime('')
+            else
+                setDateTime(f1.SearchText)
+            const f2 = props.Filter.find(f => f.Operator === '<' || f.Operator === '<=')
+            if (f2 == null)
+                setSecondDateTime('')
+            else
+                setSecondDateTime(f2.SearchText)
+        }
```

Comment:
It looks like if props.Filter got changed by some outside process, it wouldn't update the operator here. Also, if we do something strange like have a filter that has two different inequalities of '>', then we wouldn't be properly setting the operator to 'after'.

Reply:
So I added logic to set the operator. That being said if you have 2 filters with `>` it will not set the Date/Time correctly because it will use the first one matching the Operator Criteria (`>`) instead the `max`. In addition for `before` and `after` it will use `dateTime` and never `secondDateTime` so you may end up with an empty DateTime in the UI even though it does apply 2 Filters of the same type. 
I am not sure that this needs to be addressed since technically this is an internal component to `gpa-gemstone` and not exposed so truly external processes have no way of altering the filter

Reply:
It may not need to be addressed now, but it may be helpful to note the limitation in the case that someone else want's to add someone else internal to gemstone using this component.